roster/views.py

```diff
@@ -4,6 +4,12 @@ def bosses(request):
     bossesFile = open('./bosses.txt', 'r')
     data = bossesFile.read()
     list = data.split('\n')
-    print(list)
     bossesFile.close()
     return render(request, 'bosses.html', {'list': list})
+
+def bosses2(request):
+    bossesFile = open('./bosses.txt', 'r')
```

Comment:
We can use recommended for python practice and naming variables snake_case way. Instead of bossesFile here would be bosses_file